cloudbuild.yaml

```
@@ -244,7 +244,7 @@ steps:
 
 - name: 'e2e-runner'
   args:
-    - 'PlayerTracking=true&StateAllocationFilter=true&PlayerAllocationFilter=true&SDKGracefulTermination=true'
+    - 'PlayerTracking=true&PlayerAllocationFilter=true&SDKGracefulTermination=true'
```

Thanks for doing this!

```suggestion
    - 'PlayerTracking=true&StateAllocationFilter=false&PlayerAllocationFilter=true&SDKGracefulTermination=true'
```

Basically, we want to test the reverse.

Which makes me also realise that `CustomFasSyncInterval` should be `false` in there too. I'll enter that as a separate PR (unless you want to sneak it in).

This should also mean that the changes to the e2e tests below shouldn't be necessary (I don't think).

---

I see. I'll reverse the flag and revert the test if it's unnecessary.

> Which makes me also realise that CustomFasSyncInterval should be false in there too. I'll enter that as a separate PR (unless you want to sneak it in).
I could sneak it in, but I think it's better to release in 1.25.0 if still make it in time, or release in patch version like 1.25.1.
What do you think?

---

> This should also mean that the changes to the e2e tests below shouldn't be necessary (I don't think).

The changes that I made is for unit tests running by `make test-go` not e2e tests, so It seems to be necessary.

---

> I could sneak it in, but I think it's better to release in 1.25.0 if still make it in time, or release in patch version like 1.25.1.

Don't worry about it - it's a small fix, I'll take care of it.

> The changes that I made is for unit tests running by make test-go not e2e tests, so It seems to be necessary.
Oh so it was! My bad. Ignore that comment then.